index.html

```diff
@@ -0,0 +1,697 @@
+
+      <div class="d-none d-md-block order-last order-md-last my-4 p-4 p-lg-0 w-100 bg-white">
+        <div class="row border mb-3 badge-light  col-sm-6 col-lg-12 m-auto ">
+          <img class="w-25 col-2 my-3 pr-0" src="assets/nplogo.jpg" alt="Newsletter logo">
+          <h5 class="font-weight-bolder mt-3 pr-1 col-10">START YOUR DAY WITH OUR TOP 5 ARTICLES</h5>
+          <input class="border my-1 mx-3 p-1 w-100" type="email" name="email" placeholder="Email...">
+          <button type="submit" class="mt-1 p-1 mb-3 mx-3 border-0 text-white w-100 font-weight-bold bg-danger">
+            FREE SIGN-UP >
+          </button>
+        </div>
+      </div>
+
+      <div class="d-md-block order-last order-md-last card border-1 mb-4">
+        <figure class="position-relative">
+          <img src="assets/sponsor2.jpg" alt="showcase-image" class="w-100" />
+          <figcaption class="position-absolute fixed-bottom w-40 pl-1 d-flex justify-content-center align-items-center bg-light font-weight-bolder text-primary">
+            SPONSORED INSIGHT
+          </figcaption>
+        </figure>
+        <a class="mb-2 p-1 font-weight-bolder text-body h5" href="#">What Makes International Schools so Special?</a>
+        <p class="pl-1">One of the single most important decisions you make for your child is whom to entrust with their education. See why so many parents are turning to International Schools.</p>
+      </div>
+      <div class="d-md-block order-last order-md-last card border-1 mb-4">
+        <figure class="position-relative">
+          <img src="assets/sponsor3.png" alt="showcase-image" class="w-100" />
+          <figcaption class="position-absolute fixed-bottom w-40 pl-1 d-flex justify-content-center align-items-center bg-light font-weight-bolder text-primary">
+            RANKINGS
+          </figcaption>
+        </figure>
+        <a class="mb-2 p-1 font-weight-bolder text-body h5" href="#">America's Most Responsible Companies 2021</a>
+        <p class="pl-1">We partnered with Statista to rank the country's Most Responsible Companies. They share the values that we all prioritize, and that's worth recognizing in 2021</p>
+      </div>
+      <div class="d-md-block order-last order-md-last card border-1 mb-4">
+        <figure class="position-relative">
+          <img src="assets/sponsor.jpg" alt="showcase-image" class="w-100" />
+          <figcaption class="position-absolute fixed-bottom w-40 pl-1 d-flex justify-content-center align-items-center bg-light font-weight-bolder text-primary">
+            SPONSORED INSIGHT
+          </figcaption>
+        </figure>
+        <a class="mb-2 p-1 font-weight-bolder text-body h5" href="#">Hokkaido: Japan’s Northern Frontier</a>
+        <p class="pl-1">The Biden Administration will issue an order stopping construction of the border wall on its first day.
+        </p>
+      </div>
+    </div>
+
+  </section>
+
+<br>
+
+<section style="font-family: Arial, Helvetica, sans-serif;"
```

Comment:
- [ ] Please don't include inline stylings, instead put all of this in the CSS file.